crypto/asn1/a_time.c

```diff
@@ -471,13 +471,15 @@ static const char _asn1_mon[12][4] = {
 };
 
 /* returns 1 on success, 0 on BIO write error or parse failure */
-int ASN1_TIME_print(BIO *bp, const ASN1_TIME *tm)
+int ASN1_TIME_print(BIO *bp, const ASN1_TIME *tm, int date_format)
 {
-    return ossl_asn1_time_print_ex(bp, tm) > 0;
+    if (date_format == 1)
```

Comment:
Dont use magic number 1 here..

Reply:
Not sure what C best practice dictates for trying to share constants. I went with duplicating the lines:

```
# define FORMAT_RFC_822   0
# define FORMAT_ISO_8601  1
```

In this file, but it seems preferable to import these constants. Do you know where I should define them so that they can be imported in multiple places?